pkg/kubectl/cmd/apply/apply.go

```diff
@@ -56,7 +56,8 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl/validation"
 )
 
-type ApplyOptions struct {
```

Comment:
don't change these.

Reply:
Yes, these changes are not recommended, just leave them as they are.
Others LGTM.